content/en/docs/instrumentation/java/_index.md

```diff
@@ -7,6 +7,7 @@ aliases: [/java, /java/metrics, /java/tracing]
 cascade:
   vers:
     instrumentation: 1.32.0
+    instrumentation-alpha: 1.32.0-alpha
```

Comment:
I'm not sure this will work with our existing automation

https://github.com/open-telemetry/opentelemetry.io/blob/bf5ed4ff3f36ac3a1b4c78d7054741f86e85ad54/.github/workflows/auto-update-versions.yml#L30C88-L30C88

can we get away without it, or define it as `{{ instrumentation }}-alpha`?

Reply:
> I'm not sure this will work with our existing automation
> 
> https://github.com/open-telemetry/opentelemetry.io/blob/bf5ed4ff3f36ac3a1b4c78d7054741f86e85ad54/.github/workflows/auto-update-versions.yml#L30C88-L30C88
> 
> can we get away without it, or define it as `{{ instrumentation }}-alpha`?

Done